intro-HPC/ch_running_jobs_with_input.tex

```diff
@@ -295,6 +311,27 @@ \subsection{Pre-defined quotas}
 but you can never exceed the hard quota. The user will get warnings as soon as
```

Comment:
Strictly speaking, this is not true on GPFS :-D If you write sufficiently fast, you _will_ exceed the hard quota, but the writing will stop as soon as the FS detects it.